testing/trino-testing-services/src/main/java/io/trino/testng/services/ManageTestResources.java

```diff
@@ -88,7 +89,10 @@ private static boolean isEnabled()
         if (System.getProperty("ManageTestResources.enabled") != null) {
             return Boolean.getBoolean("ManageTestResources.enabled");
         }
-        if (System.getenv("DISABLE_REPORT_RESOURCE_HUNGRY_TESTS_CHECK") != null) {
+        if (getenv("DISABLE_REPORT_RESOURCE_HUNGRY_TESTS_CHECK") != null) {
+            return false;
+        }
+        if (getenv("TESTCONTAINERS_REUSE_ENABLE") != null) {
             return false;
         }
```

Comment:
nit: if branches can be collapsed to a single one in this case (same return value and same logical operation)